app-network/frp/autobuild/defines

```diff
@@ -4,3 +4,4 @@ PKGDEP="glibc"
 BUILDDEP="go"
 PKGDES="Fast reverse proxy"
 ABSPLITDBG=0
+ABPATCHLAX=yes
```

Comment:
Why not rebase the patches?

Reply:
Do you mean repatching instead of using ABPATCHLAX...?

Reply:
> Do you mean repatching instead of using ABPATCHLAX...?

Yes, please

Reply:
Ok, done